### DISCUSSION TITLE
Disable auto collapsing in Page Index?

### DISCUSSION THREAD

**Discussion by SpikyPillow** [zim-desktop-wiki/zim-desktop-wiki#2550] [Q&A] [answered]
When you switch pages in the page index it will auto collapse / open pages depending on what you select.
Example: In A > B > C, going from C to A will collapse "B" letting you only see A and B. 
There's also the auto expand: Going to B from A will expand it and allow you to see C again.

I don't like this feature (as I'd like to stay in view of my entire hierarchy) and cannot find a way to disable it. I don't mind the auto opening that much, but it stands to reason that an option to disable auto collapsing should probably be accompanied by one to disable auto expanding.

Is there a way to disable this?

**Comment by introt** [comment, marked as answer]
Hi! I believe the setting you're looking for is "Automatically collapse sections on close page", previously discussed at #2119. You can find it under Preferences > Plugins > Page Index > Configure

**Reply by SpikyPillow** [reply]
One of those moment where you look at something and read the words and yet somehow don't process what they mean...
I was convinced it wasn't disable-able since i looked all over and couldn't find it -- despite looking right at it.
Thanks!
